Sources/Resources/Sentry.modulemap

```diff
@@ -3,4 +3,24 @@ framework module Sentry {
 
     export *
     module * { export * }
+    
+    explicit module HybridSDK {
+        header "PrivateSentrySDKOnly.h"
+        header "PrivatesHeader.h"
+        header "SentryAppStartMeasurement.h"
+        header "SentryBinaryImageCache.h"
+        header "SentryBreadcrumb+Private.h"
+        header "SentryDebugImageProvider+HybridSDKs.h"
+        header "SentryDependencyContainer.h"
+//        header "SentryEnvelope.h"
+        header "SentryEnvelopeItemType.h"
+        header "SentryFormatter.h"
+        header "SentryFramesTracker.h"
+        header "SentryOptions+HybridSDKs.h"
+        header "SentryScreenFrames.h"
+        header "SentrySessionReplayIntegration-Hybrid.h"
+        header "SentrySwizzle.h"
+        header "SentryUser+Private.h"
```

Comment:
h: I think we should create an umbrella header for private headers. 
It gives us more control if necessary later.